app/controllers/api/v1/partners_controller.rb

```diff
@@ -0,0 +1,29 @@
+class Api::V1::PartnersController < ApiController
+  def create
+    partner_service = PartnerService.new
```

Comment:
Why did you create a service to do this? `devise-invitable` will handle the creation, emailing, and so forth. `Partner.invite!(...)`

Reply:
sweet. I didn't know that. Updating it.